### ISSUE TITLE
Uncaught TypeError: Cannot read property 'selectedPath' of undefined

### ISSUE THREAD

**Issue by barrytickle** [leijou/php-server#19]
I get this whenever I try to open a file in atom with the server, not sure why it's doing this now. However it never used to.

**Atom**: 1.17.0 ia32
**Electron**: 1.3.15
**OS**: Microsoft Windows 10 Home
**Thrown From**: [php-server](https://github.com/leijou/php-server) package 0.7.0

### Stack Trace

Uncaught TypeError: Cannot read property 'selectedPath' of undefined

```
At /C:/Users/Barry/.atom/packages/php-server/lib/php-server.coffee:89

TypeError: Cannot read property 'selectedPath' of undefined
    at Object.startTree (/packages/php-server/lib/php-server.coffee:58:67)
    at /packages/php-server/lib/php-server.coffee:46:70)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (~/AppData/Local/atom/app-1.17.0/resources/app/src/command-registry.js:265:1)
    at CommandRegistry.handleCommandEvent (~/AppData/Local/atom/app-1.17.0/resources/app/src/command-registry.js:3:1)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (~/AppData/Local/atom/app-1.17.0/resources/app/src/command-registry.js:166:1)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (~/AppData/Local/atom/app-1.17.0/resources/app/src/atom-environment.js:1292:1)
    at EventEmitter.outerCallback (~/AppData/Local/atom/app-1.17.0/resources/app/src/application-delegate.js:347:1)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
     -0:11.4.0 php-server:start-tree (div.header.list-item.project-root-header)
```

### Non-Core Packages

```
atom-live-server 2.1.2 
atom-material-syntax 0.4.6 
atom-material-ui 1.3.6 
atom-terminal-panel 4.4.4 
autocomplete-paths 1.0.5 
autocomplete-php 0.3.7 
br4ckets-dark 1.0.0 
br4ckets-light 1.0.0 
local-server-web-view 1.1.0 
php-server 0.7.0 
pigments 0.39.1 
```

**Comment by barrytickle** [comment]
Issue resolved, 

Atom 0.17 appears to be causing the issue with PHP server, Do not update atom past 0.16! 

https://github.com/atom/atom/releases/tag/v1.16.0

**Comment by sumnermic** [comment]
I'm encountering this issue as well after Atom has been updated to **1.17.0**.
Unfortunately, I am unable to use the package until we find a fix please @leijou ?

**Comment by ttristan** [comment]
Hi,
@barrytickle no, not resolved. Not updating is not an option... 
I have fixed the issue with #20 and hope that @leijou will merge the pull request. Otherwise you can fix this manually: Go to to [User]/.atom/packages/php-server/lib and open php-server.coffee and change lines 58 and 62 to:
`atom.packages.getLoadedPackage('tree-view').mainModule.treeView.selectedPath`

**Comment by ctocph** [comment]
Thanks @ts-de. Works perfectly good and now I can work again...

**Comment by Briansbum** [comment]
Worked perfectly, thanks @ts-de

**Comment by marc0tjevp** [comment]
@ts-de Thank you very much

**Comment by ozzro** [comment]
Indeed. Adding my thanks !! @ts-de

**Comment by zisheng-ai** [comment]
@ts-de Thanks for you and can you fork this package?

**Comment by ttristan** [comment]
I think I did back then, you can find it in my repos.

**Comment by leijou** [comment]
Sorry all for the radio silence, I've not been getting notifications from GitHub and haven't been on here in a while. Thanks especially @ts-de 
#20 has now been released with other fixes in v0.8.0
